src/cascadia/LocalTests_SettingsModel/ProfileTests.cpp

```diff
@@ -307,4 +308,22 @@ namespace SettingsModelLocalTests
         VERIFY_ARE_EQUAL(L"profile4", settings->_allProfiles.GetAt(0).Name());
     }
 
+    void ProfileTests::DuplicateProfileTest()

```

Comment:
Bonus points: add a test that specifically covers the dynamic profile case (since that one is more complex)